### DISCUSSION TITLE
Cloudfront cache policy and origin request policy

### DISCUSSION THREAD

**Discussion by mmigliari** [GoogleCloudPlatform/terraformer#1613] [Ideas]
Terraformer currently supports pulling Cloudfront distributions, however it does not also pull aws_cloudfrot_cache_policy and aws_cloudfront_origin_request_policy for a given distribution.  

Oftentimes more complex Cloudfront distributions are build using custom cache policies and origin request policies, so for a complete cloudfront solution, terraformer should also support these two resources.

Are there any plans to implement this?

**Comment by shblue21** [comment]
@mmigliari I'm not the main Terraform contributor, but I'll try to develop it and do a pull request.

**Reply by shblue21** [reply]
@mmigliari I added cache policy feature, https://github.com/GoogleCloudPlatform/terraformer/pull/1634
I checked it with my resource, but please comment if there is something wrong after the release.
